ver/us/splat.yaml

```diff
@@ -3202,7 +3454,33 @@ segments:
     subsegments:
     - [0x36A8D0, c, ending_decals]
     - [0x36AE80, data]
-  - [0x36AEE0, bin] # ending_decals gfx
+  - name: effect_gfx_ending_decals
+    dir: effects/gfx
+    type: code
+    start: 0x36AEE0
+    vram: 0x09000000
+    symbol_name_format: $VRAM_$ROM
+    subsegments:
+    - [0x36AEE0, c, ending_decals]
+    - start: 0x36AEE0
+      type: .data
+      name: ending_decals
+      subsegments:
+      - [0x36AEE0, i8, D_09000000_36AEE0, 64, 64]
+      - [0x36BEE0, ci4, D_09001000_36BEE0, 64, 32]
+      - [0x36C2E0, ci4, D_09001400_36C2E0, 64, 32, D_09001000_36BEE0]
+      - [0x36C6E0, ci4, D_09001800_36C6E0, 64, 32, D_09001000_36BEE0]
+      - [0x36CAE0, palette, D_09001000_36BEE0]
+      - [0x36CB00] # padding
+      - [0x36CCE0, vtx, D_09001E00_36CCE0]
+      - [0x36CD20, gfx, D_09001E40_36CD20]
+      - [0x36CD88, gfx, D_09001EA8_36CD88]
+      - [0x36CDF0, gfx, D_09001F10_36CDF0]
+      - [0x36CE58, gfx, D_09001F78_36CE58]
+      - [0x36CEF8, vtx, D_09002018_36CEF8]
+      - [0x36CF38, gfx, D_09002058_36CF38]
+      - [0x36CF58, gfx, D_09002078_36CF58]
+      - [0x36D018] # padding
```

Comment:
these end padding comments can be removed (I think the ones mid-subsegments are fine though)

Reply:
Do you mean removing the comment only (`# padding`) or the whole subsegment (`- [0xXXXXXX] #padding`)? Removing the whole thing seems to work without messing the previous subsegment but I'm not sure why.

Reply:
I mean to just remove the comments. it's still good to note the data ends at this spot